Gruntfile.js

```diff
@@ -203,6 +203,12 @@ module.exports = function(grunt) {
       }
     },
     exec: {
+      optimizeJs: {
+        cmd: './node_modules/.bin/optimize-js ./static/dist/inbox.js > ./static/dist/inbox.js.bak && ' +
+                                          'mv ./static/dist/inbox.js.bak ./static/dist/inbox.js && ' +
+             './node_modules/.bin/optimize-js ./static/dist/templates.js > static/dist/templates.js.bak && ' +
+                                          'mv ./static/dist/templates.js.bak ./static/dist/templates.js',
+      },
```

Comment:
Use the semi-official [grunt task](https://github.com/sergejmueller/grunt-optimize-js).